styles/style.css

```diff
@@ -0,0 +1,115 @@
+/* GLOBAL CSS */
+
+* {
+    padding: 0; 
+    margin: 0;
+}
+
+html {
+    height: 100%;
+}
+
+body {
+    color: #322;
+}
+
+.clearfix {
+    clear:both ;
+}
+
+/* HEADER AREA */
+
+#logo img {
```

Comment:
Im not too sure about conventions, but to my understanding using ID's to style is not the best practice due to IDs being way more specific than classes

Reply:
more information: https://specificity.keegan.st/